supergloo/docs/mesh/tutorials-3-prometheus-metrics.md

```diff
@@ -0,0 +1,151 @@
+---
+title: "Tutorial: Configuring Prometheus to Collect Istio Metrics"
+weight: 5
+---
+
+### Summary
+
+In this tutorial we'll take a look at how to configure [the Prometheus monitoring system](https://prometheus.io/) to scrape our meshes for metrics using SuperGloo.
+
+Monitoring the traffic being sent between a system of microservices is one of the primary features provided by service meshes. Service meshes make it easy to collect
+ metrics from a large, distributed system in a centralized metrics store such as Prometheus. Typically, when installing a mesh, the mesh and metrics store must be manually configured in order to produce readable metrics.
+ 
+SuperGloo provides features to automatically propagate metrics from a managed mesh with one or more instances of a metrics store. 
+
+Let's dive right in.
+
+### Tutorial
+
+First, ensure you've:
+
+- [installed SuperGloo](../../installation)
+- [installed Istio using supergloo](../install)
+- [Deployed the Bookinfo sample app](../bookinfo)
+
+Next, we'll need an instance of Prometheus running in our cluster. If you've already got Prometheus installed, you can skip this step.
+
+> Note: For SuperGloo to configure Prometheus correctly, it requires that the Prometheus server is configured with a 
+> [ConfigMap](https://kubernetes.io/docs/tasks/configure-pod-container/configure-pod-configmap/), and that the
+> key for the Prometheus configuration file is named `prometheus.yml` 
+
+To install a simple Prometheus instance for the purpose of this tutorial, run the following:
+
+```yaml
+kubectl create ns prometheus-test
+kubectl apply -n prometheus-test -f \
+  https://raw.githubusercontent.com/solo-io/solo-docs/master/supergloo/examples/prometheus/prometheus-demo.yaml
+```
+
+> Note: We can watch the pods get created for Prometheus with `kubectl get pod -n prometheus-test -w`
+ 
+Let's take a look at the configmap that was created by this install for us:
+
+```bash
+kubectl get configmap -n prometheus-test
+
```

Comment:
So it might be worth breaking this up into two separate blocks. The "copy and paste" feature from the docs makes it difficult to copy and paste if both the command and the output are presented together. so for example:

```bash
command in this block
```

```noop
output in this block
```